pkg/apis/operator/v1/types.go

```diff
@@ -385,6 +385,12 @@ type CNISpec struct {
 	// Default: Calico
 	// +kubebuilder:validation:Enum=Calico;GKE;AmazonVPC;AzureVNET
 	Type CNIPluginType `json:"type"`
+
+	//LogLevel specifies the log level the CNI plugin is configured to run at.
+	// Default: INFO
+	// +optional
+	// +kubebuilder:validation:Enum=ERROR;WARNING;INFO;DEBUG
+	LogLevel string `json:"log_level"`
```

Comment:
Need to think about this some more. I think for now I would recommend not including the API changes in this PR, so that they are separate from the more important change of mounting in the files. 

As an aside, this should probably be its own type, with values of format `Error,Warning,Info,Debug` (note capitalization).

Reply:
The capitalization is weird because our docs list the options in all uppercase whereas we seem to use `info` in other places. Not sure what the values really should be.

Reply:
I can remove the API for now, I don't think anyone will be changing the log level (for now) so it should be fine. I was just hoping to add the configuration just in case, but it's not entirely thought through at this point.

Reply:
Well, this is the operator API, so it doesn't matter really what the CNI plugin says, we need to be consistent within the operator API itself. 

I think it does toLower() on it anyway :D